src/Cis194/Hw/LogAnalysis.hs

```diff
@@ -6,22 +6,44 @@ module Cis194.Hw.LogAnalysis where
 import  Cis194.Hw.Log
```

Comment:
A note about a flaw in this exercise: `LogMessage` shouldn't have an `Unknown` constructor. Error handling like this should be done in parallel to the type for valid log messages. At a minimum you might have: `Either String LogMessage` or something. This causes problems later in the exercises when you have to ignore `Unknown`.